server/interpreter.js

```diff
@@ -25,30 +25,24 @@
 
 /**
  * Create a new interpreter.
- * @param {string|!Object} code Raw JavaScript text or AST.
  * @param {Function=} opt_initFunc Optional initialization function.  Used to
  *     define APIs.  When called it is passed the interpreter object and the
  *     global scope object.
  * @constructor
  */
-var Interpreter = function(code, opt_initFunc) {
-  if (typeof code == 'string') {
-    code = acorn.parse(code, Interpreter.PARSE_OPTIONS);
-  }
-  this.ast = code;
+var Interpreter = function(opt_initFunc) {
```

Comment:
What is the purpose of opt_initFunc?  If the interpreter does not commence running any code after being constructed, is this needed?